src/unicef_djangolib/templates/403.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_base.html" %}
```

Comment:
Wondering what's the use case for 403.html, how did you test the rendering is ok for this template ?